OmniBLE/Bluetooth/MessagePacket.swift

```diff
@@ -35,7 +35,7 @@ struct MessagePacket {
         let priority = f2.get(1) != 0
         let lastMessage = f2.get(2) != 0
         let gateway = f2.get(3) != 0
```

Comment:
I'm not up-to-speed with the message parsing changes but do we intend to vary from the AndroidAPS logic here? It seems your original code below matched their implementation.

Reply:
Here's the relevant unit tests which we may want to check against this before merge: https://github.com/randallknutson/OmniBLE/blob/87687785f4fd037975d5741cb939b1eeac7e8451/OmniBLETests/Driver/Comm/message/MessagePacketTests.swift

Reply:
This was purely a bug on my part. The MessageType should come from f2, not f1. I noticed it when parsing and encoding again. The messagetype was getting lost and this was the fix. If this is how android is doing it, they are doing it wrong.